kolibri/plugins/management/assets/src/state/store.js

```diff
@@ -37,7 +37,7 @@ const mutations = {
   UPDATE_USERS(state, users) {
     users.forEach(user => {
       state.pageState.users.forEach(existingUser => {
-        if (existingUser.id === user.id.toString()) {
+        if (existingUser.id === user.id) {
```

Comment:
Already a string, and if we ever decide to use an int (which makes more sense imo) this would just be something to take care of. 

@ralphiee22 and I were trying to figure out at what point it's converted to a string. Not a whole lot of progress.